src/dlls/mscordac/mscordac_unixexports.src

```diff
@@ -29,6 +29,7 @@ PAL_InitializeDLL
 PAL_TerminateEx
 PAL_IsDebuggerPresent
 PAL_ProbeMemory
+PAL_Random
```

Comment:
Why do we need to export `PAL_Random` here?

Reply:
Because the CoCreateGuid depends on it and it is now in palrt. I've moved CoCreateGuid to palrt.

Reply:
To be more precise mscordbi depends on symbols exported by mscordac and mscordbi uses CoCreateGuid, so it needs access to the underlying PAL_Random.

Reply:
ok